newsfragments/346.removal.rst

```diff
@@ -1 +1 @@
-Remove support for Python 2. trustme now requires Python>=3.6 (CPython or PyPy).
+Remove support for Python < 3.7. trustme now requires Python>=3.7 (CPython or PyPy).
```

Comment:
```suggestion
Remove support for Python 2. trustme now requires Python>=3.7 (CPython or PyPy).
```

Sorry for not replying earlier, but dropping Python 3.6 is the minor event here compared to dropping Python 2. The suggestion should clarify things!